lib/fizzy/instructions.cpp

```diff
@@ -43,20 +43,20 @@ constexpr InstructionMetrics instruction_metrics_table[256] = {
     /*                       0x19 */ {},
 
     // 5.4.2 Parametric instructions
-    /* drop                = 0x1a */ {1, -1},
-    /* select              = 0x1b */ {3, -2},
+    /* drop                = 0x1a */ {0, 0},
```

Comment:
These are changed only for consistency with number of types in `instruction_type_table` below.

In fact these values are not used anymore at all, and I plan to remove `instruction_metrics_table` in another PR, or rather transform it into the table with only memory alignment values.

Reply:
How are these change not breaking anything?

Reply:
These values are not used anymore, only types table is used for checks including stack height checks.

`drop` just drops an operand from the stack.

`select` pops one `i32` from the stack according to `instruction_type_table`, then drops one operand without type checking in this PR (to be added in #413)
I guess we don't have a unit test checking required number of operands for `select`, and spec tests for it also look not perfect, see description in #413.

Reply:
I've added a disabled test in this PR.

Reply:
I would leave the table unmodified with a comment it is obsolete.

Do I understand it right that now you can create a program that will crash because of the wrong max stack height? And that will be fixed with validation of all instructions?

Reply:
Yes, executing the disabled `select` test will crash. I believe it's only for `select`.

(I will roll back the changes to metrics.)

Reply:
I'm not saying there is a need to rollback anything. But we should crasher reproducers in tests. I can work on these as a part of the review.

Reply:
> I'm not saying there is a need to rollback anything.

I meant just the changes that don't influence anything.

Reply:
This is the test that will crash in execution https://github.com/wasmx/fizzy/blob/d039bd454a3aa902a3cf3ff532e7f1b5ebb5b920/test/unittests/validation_stack_test.cpp#L13-L25

Reply:
I came up with very similar test, pushed to the same commit.